notion-cli/notion
# Add fish support 🐟

This PR builds on the work @scalvert started in #171 to add full fish support to Notion.

- closes #171
- resolves #152

## Testing

1. Check out this branch locally. If you're using [`hub`], `hub pr checkout 266`. (If you're not using `hub`… you should do that thing. And also alias it to `git` to make your life wonderful.)

2. Create a release build: `cargo build --release`.

3. Do the equivalent of running the install script: `./dev/unix/build.sh; and ./dev/unix/install.sh`.

4. Open a new shell, or refresh your current shell with the updated config: `source ~/.config/fish/config.fish`.

[`hub`]: https://hub.github.com

Then you should be able to use Notion!
